### PR TITLE
MBM-263 feature | add myrial suit armor

### PR DESCRIPTION
**Pull request by solidwaterslayer**

### CODE REVIEW

File: src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java [hunk 1]
```diff
@@ -21,7 +22,7 @@ public enum ArmorMaterials implements ArmorMaterial {
     private final float knockbackResistance;
     private final Ingredient repairIngredient;
     
-    private ArmorMaterials(int pDurabilityMultiplier, int[] pSlotProtections, int pEnchantmentValue, SoundEvent pSound, float pToughness, float pKnockbackResistance, Ingredient pRepairIngredient) {
+    ArmorMaterials(int pDurabilityMultiplier, int[] pSlotProtections, int pEnchantmentValue, SoundEvent pSound, float pToughness, float pKnockbackResistance, Ingredient pRepairIngredient) {
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java:25]: these changes are unnecessary

File: src/main/java/com/mineboundteam/minebound/item/armor/MyrialArmorItem.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public ArmorTier getTier(){
 
     @Override
     public void registerControllers(AnimationData data) {
-        data.addAnimationController(new AnimationController<MyrialArmorItem>(this, getDescriptionId(), 20, this::predicate));
+        data.addAnimationController(new AnimationController<>(this, getDescriptionId(), 20, this::predicate));
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/armor/MyrialArmorItem.java:50]: Why did you remove this?

File: src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java [hunk 3]
```diff
@@ -5,6 +5,7 @@
 import net.minecraft.world.entity.EquipmentSlot;
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java:5]: Please rename MYRIAL_SUIT to MYRIAL_EFFIGY and then add a new ArmorMaterial for the suit; those are not the right defense values for the suit

**Review by insertStringHere — CHANGES_REQUESTED** [review]

File: src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java (hunk 1 above)
Comment (solidwaterslayer) [reply src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java]: Intellij told me to remove this.

File: src/main/java/com/mineboundteam/minebound/item/armor/MyrialArmorItem.java (hunk 2 above)
Comment (solidwaterslayer) [reply src/main/java/com/mineboundteam/minebound/item/armor/MyrialArmorItem.java]: Intellij told me to remove this.

File: src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java [hunk 4]
```diff
@@ -5,13 +5,14 @@
...
+import org.jetbrains.annotations.NotNull;
 
 /**
  * Pretty much taken from minecraft's build-in armor materials
  */
 public enum ArmorMaterials implements ArmorMaterial {
-    MYRIAL_SUIT(0, new int[]{1, 4, 5, 2}, 0, SoundEvents.ARMOR_EQUIP_CHAIN, 0f, 0f, null);
-    
+    MYRIAL_EFFIGY(0, new int[]{1, 4, 5, 2}, 0, SoundEvents.ARMOR_EQUIP_CHAIN, 0f, 0f, null),
+    MYRIAL_SUIT(15, new int[] { 2, 5, 6, 2 }, 9, SoundEvents.ARMOR_EQUIP_IRON, 0f, 0f, null);
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java:15]: Durability modifier and enchantability should be 0; this armor has custom durability and cannot be enchanted

**Comment by MattKirchh** [comment]
There was an issue with the path to the armor textures in `MyrialArmorModel`. Since it is the last night of the sprint, I am just going to push that fix to this branch and approve.

**Review by MattKirchh — APPROVED** [review]

**Review by insertStringHere — APPROVED** [review]

**Pull request merged**